README.md

```diff
@@ -506,21 +514,29 @@ missing dumps through the headnode and into Thoth.
 Running `sdc-thoth-install` as root on the headnode will install the
 latest binary on the headnode in `/opt/custom`, create a `thoth`
 user and create the necessary SMF manifest as well as a `crontab` that
-runs `sdc-thoth` in dry-run mode.  You can also download and execute
-this directly from Manta (with the obvious caveats that you should really
-never just pipe the output of `curl` to `bash` running
-as `root`):
+runs `sdc-thoth` in dry-run mode.  The latest version can be grabbed via:
+
+    # curl -k \
+      https://us-east.manta.joyent.com/thoth/public/thoth/thoth-sunos-latest.tar.gz | \
+      tar zxvf - ./opt/custom/thoth/bin/sdc-install-thoth
```

Comment:
Will fix the style. I've modified the instructions a little, but won't change the script - we need the tarball to copy over to the compute nodes.

Reply:
Yes, but we're downloading it twice. Once to extract the installer script and once done by the installer.

In the current version, the installer script and the tar are put into manta so that you can just `curl | bash` the installer which does the download for you.